.travis.yml

```diff
@@ -1,4 +1,7 @@
-dist: xenial
+dist: bionic
```

Comment:
I assume we're updating the distribution because `xenial` doesn't support ppc64le?

Reply:
xenial supports ppc64le but has the older (9.x) version of PostgreSQL, where we need 10.x